languages/portuguese/arquivos/DIVULGACAO.md

```diff
@@ -1 +1,38 @@
-# Divulgação
\ No newline at end of file
+# Divulgação
+> Aqui estão algumas instruções e dicas de como realizar uma boa divulgação
+
+Divulgação acaba sendo deixada por último na *maioria* das vezes, mas ela é tão importante quanto as palestras ou o público. Sem uma boa divulgação, não há público.
+
+
+> O contéudo contido nessa seção pode e deve ser constantemente atualizado. Sinta-se a vontade para contribuir. 
+
+
+
+## Onde fazer divulgação
+
+Como foi falado nos tópicos anteriores é preciso definir um local. Esses locais usualmente são universidades/empresas e queremos que as pessoas que convivem nesses locais vá para o encontro do PUG que ocorrerá. Mas isso, na maioria das vezes, não é o que ocorre. E por que isso não ocorre? Porque a divulgação ficou restrita aos locais usuais utilizados, acomodou-se, e não atingiu novas pessoas. 
+
+
+Como tentamos sair da nossa zona de conforto e atingir pessoas novas?
+A seguir, ofereceremos algumas dicas para lhe ajudar.
+
+1. Lista de email do local que sediará o evento;
+2. Compartilhar em newsletter de eventos na região (ex. [Manguezal](https://comunidademanguezal.slack.com/))
+3. Compartilhar em grupos relacionados de Telegram, WhatsApp e outros
+4. Utilizar o Telegram/Twitter (caso você não tenha acesso, entre em contato com os adms)
+5. Lista de email de pessoas chaves de empresa, universidades e outras comunidades
+5. Compartilhar por outras fontes, que forem vistas como necessárias
+
+
+## Como fazer divulgação
+
+Após ter definido os locais onde seriam realizadas as divulgações, está na hora de definir como fazer a divulgação. E tudo se inicia com uma arte bem feita e chamativa.
+
+**Você não é um designer? Você não conhece um designer?** Tudo bem, existem ferramentas onlines muito boas para compensar nossa falta de dotes artisticos. Uma delas é o [Canva](https://www.canva.com/), com ele você consegue criar folders a partir de templates prontos. 
+
+Mas o que você quer é dicas, certo? Utilizar, por exemplo, o Canva não é tão difícil. A dificuldade é como fazer o contéudo contido nele. Vamos lá!!
+
+- Se for inserir imagens ao template, utilize imagens *png*. Normalmente, não possuem fundo e são perfeitas para fazer arte.
+- Preste atenção na resolução. Cada rede social possuí sua resolução favorite, se atente a isso.
+- Insira imagens que signifiquem muito. Por exemplo, logo da python, do telegram, de universidade, de twitter.
+- Insira data, local, links encurtados.
```

Comment:
O quê? Quando? Onde? Porquê? Como? Acho importante que quando possível responder essas perguntas quando for fazer alguma divulgação ou texto. Tenta colocar isso no texto.

Reply:
Esse PR foi só um rascunho, vai ser detalhado mais na frente, eu ia abrir uma issue disso.

Reply:
Beleza!